packages/lesswrong/server/repos/ElectionCandidatesRepo.ts

```diff
@@ -26,4 +27,23 @@ export default class ElectionCandidatesRepo extends AbstractRepo<DbElectionCandi
         "tagId" IS NOT NULL
     `, [electionName, electionTagId]);
   }
+
+  async getAmountRaised(electionName: string): Promise<ElectionAmountRaised> {
+    const result = await this.getRawDb().oneOrNone<ElectionAmountRaised>(`
+      SELECT
+        SUM(CASE WHEN "isElectionFundraiser" = TRUE THEN "amountRaised" ELSE 0 END) as "raisedForElectionFund",
+        SUM(CASE WHEN "isElectionFundraiser" = TRUE THEN "targetAmount" ELSE 0 END) as "electionFundTarget",
+        SUM("amountRaised") as "totalRaised",
```

Comment:
Is this definitely meant to include the amount for the election fundraiser? (It's very possible the answer is yes - I'm not sure what the desired behaviour is)

Reply:
This was a deliberate choice, my reasoning is that it's "technically correct" (i.e. counting all money that is a result of the donation election), and also that the total for the individual fundraisers might be unimpressively low